release-notes.hbs.md

```diff
@@ -28,7 +28,8 @@ This release includes the following platform-wide enhancements.
   Tanzu Application Platform components are behaving as expected.
 
 - [AWS Services](aws-services/about.hbs.md): Provides a more streamlined approach for integrating
-  services from AWS into Tanzu Application Platform.
+  services from AWS into Tanzu Application Platform. Currently supports RDS PostgresSQL and MySQL. The package
+  is optional and depends on `crossplane.tanzu.vmware.com`.
```

Comment:
**thought** it also depends on services-toolkit however I'm not sure we need to mention the dependency here as stk/crossplane are installed in all default profiles.

Reply:
I wasn't sure either, but it was mentioned in the Google doc. Happy to remove it! Do we want to keep mentioning the optionality? I couldn't find any other similar reference in the file, so I'm not sure it's customary.

Reply:
Done!